src/modules/aidoku/DeepLink.ts

```diff
@@ -0,0 +1,27 @@
+import { Manga } from "./Manga";
+import { Chapter } from "./Chapter";
+import { create_deeplink } from "./aidoku";
+
+/**
+ * Directly open mangas from aidoku://:sourceId/:mangaId URLs.
```

Comment:
DeepLinks parse the source's website URL to return the manga, not the aidoku:// links, which Aidoku can handle itself.

```suggestion
 * Directly open manga from source website URLs.
```